nfc/libs/NfcCoreLib/lib/LibNfc/phLibNfc_Hci.c

```diff
@@ -1149,6 +1149,16 @@ NFCSTATUS phLibNfc_HciChildDevInitComplete(void* pContext, NFCSTATUS status, voi
             {
                 PH_LOG_LIBNFC_INFO_STR("NFCEE initialization failed");
                 pLibCtx->tSeInfo.bSeState[bIndex] = phLibNfc_SeStateInvalid;
+
+                /* Following NCI 2.0 Figure 25: NFCEE State Transitions,
+                   on NFCEE_TRANSMISSION_ERROR, the NFCEE is to be considered unresponsive.
+                   We can remove it from the list. */
+                if (status == PH_NCINFC_STATUS_NFCEE_TRANSMISSION_ERROR)
```

Comment:
As per Nci1.0 specification, a NFCEE activation failure may not ends with NFCEE_TRANSMISSION_ERROR status code.
As a consequence, in NCI1.x, the NFCEE status may not be reflected as invalid in case of error.